crates/teloxide-core/src/types/message.rs

```diff
@@ -40,6 +40,11 @@ pub struct Message {
     /// forwarded to the discussion group
     pub sender_chat: Option<Chat>,
 
+    /// If the sender of the message boosted the chat, the number of boosts
+    /// added by the user
+    // Yes, it's i32 ¯\_(ツ)_/¯: https://github.com/tdlib/td/blob/cb164927417f22811c74cd8678ed4a5ab7cb80ba/td/telegram/MessagesManager.h#L1004
+    pub sender_boost_count: Option<i32>,
```

Comment:
Probably it should be in `MessageCommon`, [telegraf](https://github.com/telegraf/types/blob/7e3ecbf68b2ab99f2fd6bce510c72db0732cb502/message.ts#L26) do this